test/performance/skanetrafiken/speed-test-config.json

```diff
@@ -0,0 +1,28 @@
+{
+  "testDate": "2022-10-10",
+  "feedId": "ST",
+  "graph": "graph.obj",
+
+  "walkSpeedMeterPrSecond": 1.38,
+  "tuningParameters": {
+    "dynamicSearchWindow": {
+      "dynamicSearchWindow": {
+        "maxWinTimeMinutes": 1440
+      },
+      "stopTransferCost": {
+        "DISCOURAGED": 3000,
+        "ALLOWED": 150,
+        "RECOMMENDED": 60,
+        "PREFERRED": 0
+      }
+    },
+    "routingDefaults": {
+      "transferSlack": 180,
+      "waitReluctance": 0.175,
+      "walkReluctance": 5,
+      "maxDirectStreetDurationSeconds": 3700,
+      "unpreferredModeCost": "720 + 0.0 x",
```

Comment:
This is not currently merged in OpenTripPlanner - should we include this ?

Reply:
I removed it. Looks like API for this feature will change anyway.

Reply:
nvm, we already have configuration for cost without any coefficient. I've added it.